BioXASAcquaman.pro

```diff
@@ -5,8 +5,9 @@
 TEMPLATE = subdirs
 SUBDIRS += \
 	Initialize.pro \
-	BioXASSideAcquaman.pro \
 	BioXASMainAcquaman.pro \
-	BioXASImagingAcquaman.pro \
+	BioXASMainTableCoordinator.pro \
```

Comment:
I agree, it seems like a good idea to have the BioXAS stuff in BioXASAcquaman.pro instead of acquaman_suite.pro. It might be advantageous to have the table coordinators with each specific project if it makes sure they are built along with everything else when bioxasside_beamline.pro and bioxasmain_beamline.pro are built. Do you know if that's the case here?